src/ParcelRegistry.Importer.Console/Program.cs

```diff
@@ -22,7 +22,18 @@ private static void Main(params string[] args)
         {
             var configureForBuildingRegistry = JsonSerializerSettingsProvider.CreateSerializerSettings().ConfigureForPerceelregister();
             JsonConvert.DefaultSettings = () => configureForBuildingRegistry;
-            var settings = new SettingsBasedConfig();
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: false)
+                .AddJsonFile($"appsettings.{Environment.MachineName.ToLowerInvariant()}.json", optional: true, reloadOnChange: false)
+                .AddEnvironmentVariables()
+                .AddCommandLine(args ?? new string[0])
+                .Build();
+
+            var crabConnectionString = configuration.GetConnectionString("CRABEntities");
+            Func<CRABEntities> crabEntitiesFactory = () => new CRABEntities(crabConnectionString);
```

Comment:
```suggestion
            Func<CRABEntities> crabEntitiesFactory = () =>
            {
                var factory = new CRABEntities(crabConnectionString);
                factory.Database.CommandTimeout = 60 * 60;
                return factory;
            };
```

Can we do this? Than you can remove the CommandTimeout at CrabQueries.cs